tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot

```diff
@@ -44,20 +44,30 @@ Test Metric "Notebook CPU Usage" On ODS Prometheus
     Should Not Be Equal    ${cpu_usage_before}    ${cpu_usage_after}
 
 Test Metric "Rhods_Total_Users" On ODS Prometheus
-    [Documentation]    Verifies that metrics rhods_total_users and jupyterhub_total_users have the same value in Prometheus
+    [Documentation]    Verifies that metric value for rhods_total_users
+    ...    matches the value of its corresponding expression
     [Tags]    Sanity
     ...       Tier1
     ...       ODS-628
-    ${expression} =    Set Variable    rhods_total_users&step=1  #step = 1 beacuase it will return value of current second
-    ${rhods_total_users} =    Prometheus.Run Query    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}    ${expression}
-    Log    rhods_total_users: ${rhods_total_users.json()["data"]["result"][0]["value"][-1]}
-    ${expression} =    Set Variable    jupyterhub_total_users&step=1  #step = 1 beacuase it will return value of current second
-    ${jupyterhub_total_users} =    Prometheus.Run Query    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}
+
+    # Note: the expression ends with "step=1" to obtain the value for current second
+    ${expression} =    Set Variable    rhods_total_users&step=1
+    ${rhods_total_users} =    Prometheus.Run Query    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}
```

Comment:
Maybe $RHODS_PROMETHEUS_URL and $RHODS_PROMETHEUS_TOKEN could be added by default in the keyword so that we don't have to add those fixed values every time we call it

Reply:
I did it like this on purpose, as I wanted the library to be generic and not RHODS specific. 

Also, in ML Pipelines they will introduce another prometheus pod that will have a different url and token